RecoParticleFlow/PFClusterProducer/interface/PFRecHitQTests.h

```diff
@@ -381,6 +381,87 @@ class PFRecHitQTestHOThreshold : public PFRecHitQTestBase {
 };
 
 //
+//  Quality test that checks threshold as a function of ECAL eta-ring
+//
+#include "Calibration/Tools/interface/EcalRingCalibrationTools.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+class PFRecHitQTestECALMultiThreshold : public PFRecHitQTestBase {
+ public:
+  PFRecHitQTestECALMultiThreshold() {
+
+  }
+  
+ PFRecHitQTestECALMultiThreshold(const edm::ParameterSet& iConfig):
+  PFRecHitQTestBase(iConfig)
+  { 
+    thresholds_ = iConfig.getParameter<std::vector<double> >("thresholds");
+  }
+
+  void beginEvent(const edm::Event& event,const edm::EventSetup& iSetup) {
```

Comment:
indentation in this (large) modified part differs from 91X version. Better sync up.